### PR TITLE
{devel}[foss/2021a] torchsampler v0.1.2 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
Test report by @smoors
**FAILED**
Build succeeded for 2 out of 6 (2 easyconfigs in total)
node304.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/1eb7b07f0df5a1385c1eb6fe8636168a for a full test report.

File: easybuild/easyconfigs/t/torchsampler/torchsampler-0.1.2-foss-2021a.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonBundle'
+
+name = 'torchsampler'
+version = '0.1.2'
+
+homepage = 'https://github.com/ufoym/imbalanced-dataset-sampler'
+description = """
+A (PyTorch) imbalanced dataset sampler for oversampling low classesand
```
Comment (akesandgren) [inline easybuild/easyconfigs/t/torchsampler/torchsampler-0.1.2-foss-2021a.eb:8]: missing " " before last "and"

File: easybuild/easyconfigs/t/torchsampler/torchsampler-0.1.2-foss-2021a-CUDA-11.3.1.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+homepage = 'https://github.com/ufoym/imbalanced-dataset-sampler'
+description = """
+A (PyTorch) imbalanced dataset sampler for oversampling low classesand
+undersampling high frequent ones."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('CUDA', '11.3.1', '', True),
```
Comment (akesandgren) [inline easybuild/easyconfigs/t/torchsampler/torchsampler-0.1.2-foss-2021a-CUDA-11.3.1.eb:16]: Change to use SYSTEM instead of True

**Review by akesandgren — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/t/torchsampler/torchsampler-0.1.2-foss-2021a-CUDA-11.3.1.eb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonBundle'
+
+name = 'torchsampler'
+version = '0.1.2'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://github.com/ufoym/imbalanced-dataset-sampler'
+description = """
+A (PyTorch) imbalanced dataset sampler for oversampling low classesand
```
Comment (akesandgren) [inline easybuild/easyconfigs/t/torchsampler/torchsampler-0.1.2-foss-2021a-CUDA-11.3.1.eb:9]: Missing " " before "and" here too

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
closing in favor of #16464

**Pull request closed without merging**